WalletWasabi.Gui/Controls/WalletExplorer/SendTabViewModel.cs

```diff
@@ -385,9 +385,9 @@ public SendTabViewModel(WalletViewModel walletViewModel, bool isTransactionBuild
 							}
 							signedTransaction = signedPsbt.ExtractSmartTransaction(result.Transaction);
 						}
-						catch (Exception ex)
+						catch (Exception)
 						{
-							NotificationHelpers.Error(ex.ToTypeMessageString());
+							NotificationHelpers.Error("Could not find the device. Make sure it is connected.");
```

Comment:
```suggestion
							NotificationHelpers.Error("Could not find the hardware wallet. Make sure it is connected.");
```

Reply:
More things can happen here. It depends on the exception.